docs/example_usage/breeds_datasets.rst

```diff
@@ -0,0 +1,288 @@
+Creating BREEDS subpopulation shift benchmarks
+===============================================
+
+In this document, we will discuss how to create BREEDS datasets, used in 
+the `code <https://github.com/MadryLab/BREEDS_benchmarks>`_ `release
+<https://github.com/MadryLab/BREEDS_benchmarks>`_ based on our paper [STM20]_. 
+These datasets can be used to study model robustness to subpopulation 
+shift---a specific form of distribution shift wherein the subpopulations of data 
+(e.g., animal breeds) present in the training set (source domain) are 
+entirely different from the test set (target domain). 
```

Comment:
Give an example of a dog vs cat classifier: e.g. "On the train set we are classifying between tabby cats and labradors, and in the test set we are classifying between Egyptian cats and golden retrievers." It's kind of hard right now to tell exactly what the dataset is early on in the docs